test/tools/WebListener/WebListener.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.0.0-preview5-19227-01" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.0.0-preview6.19307.2" />
```

Comment:
I think we should update all packages to latest versions. (As soon as we can compile the PR and pass tests.)

Reply:
Which other packages? This is the only package that I was aware of that is tightly coupled to the runtime

Reply:
All. Packages we use come from .Net Core 2.*. We need to update to latest preview 3.0.

Reply:
OK, thanks for the reference, I updated all those NuGet packages in a 2nd branch and ran the CI tests and there were only 2 failures on Windows, if you want I could add this update to this PR (especially since the packages are preview6 specific)? Here are the 2 failures:
https://christophbergmeister.visualstudio.com/PowerShell/_build/results?buildId=896&view=ms.vss-test-web.unified-test-results-tab

Reply:
I propose to update step by step (package by package).

Reply:
@iSazonov  I found the culprit that contains the breaking change that causes the 2 test failures:
` System.ServiceProcess.ServiceController`. Therefore we can probably update all NuGet packages except this one and `Microsoft.Windows.Compatibility` (because it depends on `System.ServiceProcess.ServiceController`). I am running builds at the moment in my fork to prove this theory. Should I also update the packages in `tool\packages\....` (that I think are used to create the NuGet packages for SMA, etc.)? Should the update of possible NuGet packages be in this PR or in a follow up PR (that I'd base on this PR?)?